components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/APIProvider.java

```diff
@@ -1783,6 +1783,6 @@ Mediation updateApiSpecificMediationPolicyContent(String apiId, Mediation mediat
      * @param apiRevisionId API Revision UUID
      * @throws APIManagementException if failed to delete APIRevision
      */
-    void deleteAPIRevision(String apiId, String apiRevisionId) throws APIManagementException;
+    void deleteAPIRevision(String apiId, String apiRevisionId, String organizationId) throws APIManagementException;
```

Comment:
Could not find these methods are called with the new implementation with new organizationId addition?
Are they missed to add to the PR?

Also, I hope these interface changes can be added to the master branch as well.

Reply:
Yes. These changes will be added to the master